src/librustdoc/lib.rs

```diff
@@ -645,7 +645,7 @@ where R: 'static + Send,
         paths.add_path(s, ErrorOutputType::default());
     }
     let mut cfgs = matches.opt_strs("cfg");
-    cfgs.push("rustdoc");
+    cfgs.push("rustdoc".to_string());
```

Comment:
Can you please squash this into the patch that introduced these lines?

Reply:
Done!